nvtabular/column_selector.py

```diff
@@ -93,7 +90,8 @@ def __rshift__(self, other):
         return nvtabular.ColumnGroup(self) >> other
 
     def __eq__(self, other):
-        if isinstance(other, ColumnSelector):
-            if other._names == self._names and other.subgroups == self.subgroups:
-                return True
+        if not isinstance(other, ColumnSelector):
+            return False
+        if other._names == self._names and other.subgroups == self.subgroups:
```

Comment:
I think here you can return the value of the expression `other._names == self._names and other.subgroups == self.subgroups` directly, since this returns `True` when the expression is `True` and `False` when the expression is `False`.